content/en/blog/_posts/2023-01-17-confidential-kubernetes.md

```diff
@@ -0,0 +1,155 @@
+---
+layout: blog
+title: "Confidential Kubernetes: Use Confidential Virtual Machines and Enclaves to improve your cluster security."
+date: 2023-01-17
+slug: "confidential-kubernetes"
+---
+
+**Authors:** Fabian Kammel (Edgeless Systems), (more to add...)
+
+
+# Confidential Kubernetes: Use Confidential Virtual Machines and Enclaves to improve your cluster security.
+
+In this blog post, we will introduce the concept of Confidential Computing (CC) to improve the security and privacy properties of any computing environment. Further, we will show how the Cloud-Native ecosystem, and Kubernetes in particular, can benefit from the new compute paradigm.
+
+## Intro
```

Comment:
You don't need to have a heading to say that you're introducing the article; this can be implied. I would remove this heading or change the wording.